File: projects/1/kaitlyn_pearson/hello_name.py
import sys

def hello_name(name, num_repeat):
    for i in range(int(num_repeat)):
        print(f'{i+1}. Hello {name}!')

if __name__ == '__main__':
    if len(sys.argv) != 3:
        print('Error: Incorrect number of arguments\nUsage: python hello_name.py <name> <num_repeat>')
    elif not sys.argv[2].isdigit():
        print('Error: <num_repeat> must be a positive integer')
    else:
        name = sys.argv[1]
        num_repeat = sys.argv[2]
        hello_name(name, num_repeat)

File: projects/3/xavier_baines/lectures/5/HelloName.java
public class HelloName {
    public static void main(String[] args) {
        if (args.length != 2) {
            System.out.println("Usage: java HelloName <name> <number>");
            return;
        }

        String name = args[0];
        int count;

        try {
            count = Integer.parseInt(args[1]);
        } catch (NumberFormatException e) {
            System.out.println("Error: The second argument must be a valid integer.");
            return;
        }

        for (int i = 0; i < count; i++) {
            System.out.println("Hello " + name);
        }
    }
}
